src/System.Private.ServiceModel/tools/test/SelfHostWcfService/CertificateResources/UserCertificateResource.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using WcfTestBridgeCommon;
+
+namespace WcfService.CertificateResources
+{
+    internal class UserCertificateResource : EndCertificateResource
```

Comment:
Nit: it would be useful to add some comments describing what each CertificateResource class is for